docs/modules/ROOT/pages/references/architecture/control-api-team.adoc

```diff
@@ -0,0 +1,19 @@
+= {controlapi}: Team
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_manage_teams["Manage Teams"] features.
+
+== Object
+
+[source,yaml]
+----
+apiVersion: appuio.io/v1
+kind: Team
+metadata:
+  name: myteam1
+  namespace: org-acme-corp
+spec:
+  userRefs: <1>
+  - max-muster
+  - demo-user
+----
+<1> References to one or more `User` objects
```

Comment:
In this page I'm missing how permissions are granted or what the purpose of this team is.
Should we specify the permissions in this spec? (the API server would then create RBAC resources in the background)
Or should users manage the RBAC rules themselves? (we should mention this here then)